app/domain/authentication/security/validate_account_exists.rb

```diff
@@ -4,8 +4,6 @@ module Authentication
 
   module Security
 
-    Err ||= Errors::Authentication::Security
```

Comment:
there are other places in the codebase that still have `Err ||=` . Was that intentional?

Reply:
i didn't address this in the current PR